Objects/codeobject.c

```diff
@@ -1348,25 +1330,21 @@ code_dealloc(PyCodeObject *co)
         PyMem_Free(co_extra);
     }
 
-    Py_XDECREF(co->co_code);
     Py_XDECREF(co->co_consts);
     Py_XDECREF(co->co_names);
     Py_XDECREF(co->co_localsplusnames);
     Py_XDECREF(co->co_localspluskinds);
-    Py_XDECREF(co->co_varnames);
-    Py_XDECREF(co->co_freevars);
-    Py_XDECREF(co->co_cellvars);
     Py_XDECREF(co->co_filename);
     Py_XDECREF(co->co_name);
     Py_XDECREF(co->co_qualname);
     Py_XDECREF(co->co_linetable);
     Py_XDECREF(co->co_endlinetable);
     Py_XDECREF(co->co_columntable);
     Py_XDECREF(co->co_exceptiontable);
-    if (co->co_weakreflist != NULL)
+    if (co->co_weakreflist != NULL) {
         PyObject_ClearWeakRefs((PyObject*)co);
-    if (co->co_quickened) {
-        PyMem_Free(co->co_quickened);
+    }
+    if (co->co_warmup == 0) {
         _Py_QuickenedCount--;
```

Comment:
I think we can remove `_Py_QuickenedCount` now. IIRC, it exists only to account for extra allocation and stop the refleak checker from complaining.

No need to block this PR though, and it might be a useful counter to have anyway.
If we do keep it, we should probably move it to the stats.

Reply:
Added to https://github.com/faster-cpython/ideas/issues/310